backends/mongo.go

```diff
@@ -117,7 +143,29 @@ func NewMongo(authOpts map[string]string, logLevel log.Level, hasher hashing.Has
 	}
 
 	if m.withTLS {
-		opts.TLSConfig = &tls.Config{}
+		log.Infof("mongo backend: tls enabled")
+		opts.TLSConfig = &tls.Config{
```

Comment:
Could you add new empty lines here and between following `if` checks to aid readability?